Platform/MeteorlakeBoardPkg/Library/Stage2BoardInitLib/LowPowerSupport.c

```diff
@@ -170,9 +170,9 @@ UINT64 GetLowPowerS0IdleConstraint(VOID)
       PepConfigData->PepVmd = 0;
     }
 
-    if (PciRead16 (PCI_SEGMENT_LIB_ADDRESS (0, SA_PEG_BUS_NUM, SA_PEG1_DEV_NUM, SA_PEG0_FUN_NUM, PCI_VENDOR_ID_OFFSET)) == 0xFFFF) {
-      PepConfigData->PepPcieDg = 0;
-    }
+    // if (PciRead16 (PCI_SEGMENT_LIB_ADDRESS (0, SA_PEG_BUS_NUM, SA_PEG1_DEV_NUM, SA_PEG0_FUN_NUM, PCI_VENDOR_ID_OFFSET)) == 0xFFFF) {
+    //   PepConfigData->PepPcieDg = 0;
+    // }
```

Comment:
Remove it if it is not required.

Reply:
removed commented out lines in latest push